nifi-nar-bundles/nifi-oraclecdc-bundle/nifi-oraclecdcservice-api/.gitignore

```diff
@@ -0,0 +1 @@
+/bin/
```

Comment:
Shouldn't this be part of the https://github.com/apache/nifi/pull/3306/files pull request?